MovieGallery/Pages/OurTeam.cshtml

```diff
@@ -28,24 +28,24 @@
             <div class="card-body" style="background-color:white">
                 <h2 class="card-title">Mohini Chakravarty</h2>
                 <p class="card-text">
-                    Scrum Master
+                    <div> Scrum Master </div> 
```

Comment:
Maintaining uniformity in the code as uniformity is very important in code review.

Reply:
Is this div styled?  It doesn't appear to be serving a purpose.  If that's the case, it is clutter, and should not be present.